src/accelerate/tracking.py

```diff
@@ -417,9 +417,9 @@ def __init__(self, run_name: str, **kwargs):
         super().__init__()
         self.run_name = run_name
 
-        from comet_ml import Experiment
+        from comet_ml import start
```

Comment:
The way we should do this is via an import check. Do we know if it's `3.47.6` that introduced this? Then based on the version either use `Experiment`/`end` or `start`/`flush`

Reply:
We released the last fixes to `comet_ml.start` in the version 3.41.0 back in April 2024 (https://pypi.org/project/comet-ml/3.41.0/). I have considered using `Experiment/start` based on the package version and while the new API is 99% compatible, there is new accepted by `start` (like the experiment name or tags). That could make documenting the callback slightly more complicated.
I would personally prefer to set the minimum version of comet to `3.41.0` as it would make it clearer for end-user what parameters they can pass. But it's only a small preference, I'm also fine with having a different behavior based on the comet version.